src/Core/src/Handlers/Layout/LayoutHandler.Android.cs

```diff
@@ -47,7 +47,7 @@ public void Add(IView child)
 			_ = VirtualView ?? throw new InvalidOperationException($"{nameof(VirtualView)} should have been set by base class.");
 			_ = MauiContext ?? throw new InvalidOperationException($"{nameof(MauiContext)} should have been set by base class.");
 
-			NativeView.AddView(child.ToNative(MauiContext), 0);
+			NativeView.InsertView(child, 0, MauiContext);
```

Comment:
Why is this an insert? Shouldn't the child be added at the end?

Reply:
Yea, I wasn't really sure why the initial code had AddView(view, 0) 

So I just left it alone for now, but if it seems weird to you as well I'll just change it :-)